.github/scripts/check-for-auto-generated-changes.sh

```diff
@@ -7,9 +7,9 @@ go run pkg/codegen/main.go
 go run ./pkg/codegen crds ./charts/fleet-crd/templates/crds.yaml > /dev/null
 
 if [ -n "$(git status --porcelain)" ]; then
-    echo "Generated files have either been changed manually or were not updated.\n"
+    printf "Generated files have either been changed manually or were not updated.\n"
```

Comment:
No need for printf. Would prefer 
```suggestion
    echo "Generated files have either been changed manually or were not updated."
    echo
```